examples/callback_passthrough.cpp

```diff
@@ -1,3 +1,9 @@
+// Copyright (c) 2017-2019, University of Cincinnati, developed by Henry Schreiner
+// under NSF AWARD 1414736 and by the respective contributors
```

Comment:
```suggestion
// under NSF AWARD 1414736 and by the respective contributors.
```

(Since it sounds like it's easy for you to make this sort of change ATM)

Reply:
Good point (pun intended), done

Reply:
😝